django_project/geosight/data/models/dashboard/dashboard.py

```diff
@@ -85,6 +87,12 @@ class Dashboard(
         default='geometry_code'
     )
 
+    thumbnail = models.ImageField(
```

Comment:
Hi @zamuzakki 
You don't need to create new field here, as it make not consistence
We just directly use icon field, save the image to /thumbnails